<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiBookInfo</class>
 <widget class="QMainWindow" name="GuiBookInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="lInfo">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Book Information</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="lId">
        <property name="text">
         <string>Id: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lTitle">
        <property name="text">
         <string>Title:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lAuthor">
        <property name="text">
         <string>Author:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lCategory">
        <property name="text">
         <string>Catagory:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lPublisher">
        <property name="text">
         <string>Publisher:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lYear">
        <property name="text">
         <string>Year:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lExit">
        <property name="text">
         <string>Exist:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="butClose">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
